modules/core/keeper/msg_server.go

```diff
@@ -118,40 +90,17 @@ func (k Keeper) SubmitMisbehaviour(goCtx context.Context, msg *clienttypes.MsgSu
 		return nil, sdkerrors.Wrap(err, "failed to process misbehaviour for IBC client")
 	}
 
-	ctx.EventManager().EmitEvent(
-		sdk.NewEvent(
-			clienttypes.EventTypeSubmitMisbehaviour,
-			sdk.NewAttribute(clienttypes.AttributeKeyClientID, msg.ClientId),
-			sdk.NewAttribute(clienttypes.AttributeKeyClientType, misbehaviour.ClientType()),
-		),
-	)
-
 	return &clienttypes.MsgSubmitMisbehaviourResponse{}, nil
 }
 
 // ConnectionOpenInit defines a rpc handler method for MsgConnectionOpenInit.
 func (k Keeper) ConnectionOpenInit(goCtx context.Context, msg *connectiontypes.MsgConnectionOpenInit) (*connectiontypes.MsgConnectionOpenInitResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	connectionID, err := k.ConnectionKeeper.ConnOpenInit(ctx, msg.ClientId, msg.Counterparty, msg.Version, msg.DelayPeriod)
-	if err != nil {
+	if _, err := k.ConnectionKeeper.ConnOpenInit(ctx, msg.ClientId, msg.Counterparty, msg.Version, msg.DelayPeriod); err != nil {
```

Comment:
Same question as with the `clientID` returned by `CreateClient`.

Reply:
Same answer as above